doc/quickstart/istsos_quickstart.rst

```diff
@@ -30,111 +33,78 @@ Open a terminal and execute the following commands:
     sudo -u postgres psql -d istsos -c 'CREATE EXTENSION postgis'
 
 
-Setting up an istSOS service instance
-================================================================================
+Set up an istSOS service instance
+=================================
 
 With istSOS you can organize your sensor data in different instances. Every instance has its own database schema independent from other instances. You can even deploy other databases over your network according to your needs.
 
-
-
 The first steps into istSOS setup is to configure the default configuration options. These options will then be automatically used for your convenience by every new istSOS instance created.
 
-**1) Open the Web Admin interface**
-
-Open a browser and go to `<http://localhost/istsos/admin>`_
+#. Click :menuselection:`Geospatial --> Webservices --> istSOS` to open the Web Admin interface. Alternatively, open a browser and go to `<http://localhost/istsos/admin>`_
 
-**2) Configure your database connection**
+#. Click the :guilabel:`Enter` button in the administration page.
 
-From the toolbar buttons menu press the "Database" button and fill in the database configuration options:
+#. Configure your database connection. From the toolbar buttons menu, press the "Database" button and fill in the database configuration options:
 
-::
+   ::
 
     user: postgres
     password: postgres
     host: localhost
     port: 5432
     DB name: istsos
 
-**3) Configure the default Service provider information**
-
-
-Press the "Service provider" button and fill it using your institutional information.
+#. Press the "Service provider" button and fill it using your institutional information.
 
 
-**4) Configure the default Service identification information**
+#. Press the "Service identification" button and fill up with custom metadata that describe the service usage.
 
+#. Press the "Coordinates system" button to manage the Coordinate Reference System (CRS).
 
-Press the "Service identification" button and fill up with custom metadata that describe the service usage.
+   * *Default EPSG code*: this will be the native Coordinate Reference System of your geometries in the database.
 
-**5) Configure your service coordinates system**
+   * *Permitted EPSG*: here you can configure which other Coordinate Reference Systems your system will support and reproject if requested.
 
-Press the "Coordinates system" button to manage the Coordinate Reference System (CRS).
-
-*Default EPSG code*: this will be the native Coordinate Reference System of your geometries in the database.
-
-*Permitted EPSG*: here you can configure which other Coordinate Reference Systems your system will support and reproject if requested.
-
-**6) Configure your getobservation request options**
-
-From the toolbar buttons menu press the "GetObservation Configuration" button to
+#. From the toolbar buttons menu, press the "GetObservation Configuration" button to
```

Comment:
This should be auto-numbered but it is showing the hash symbol. @geographika can you help? I tried fixing the indent above.